lib/ditto.ts

```diff
@@ -22,7 +22,16 @@ type Command =
   | "generate-suggestions"
   | "replace";
 
-const COMMANDS = [
+interface CommandConfig<T extends Command | "add" | "remove"> {
```

Comment:
Making types here more strict but specifying `"add" | "remove"` isn't ideal. Open to improvements.

Reply:
I think there's ways we could clean this up, but probably not worth doing right now - this looks fine to me atm